### DISCUSSION TITLE
天翼云网盘，不能同时挂载个人云和家庭云吗？

### DISCUSSION THREAD

**Discussion by wuliangren** [AlistGo/alist#7294] [Q&A]
个人云识别正常，家庭云识别不出来

**Comment by GitCourser** [comment]
我个人家庭同时挂正常啊

**Comment by LINJIAXIN-China** [comment]
![微信图片_20241023230437](https://github.com/user-attachments/assets/595d71c5-25f9-449f-814d-069028b48095)
可以的，要分别挂载两个目录，像我这样
